src/main/webapp/newDrug.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>New drug</title>
+</head>
+<body>
+    <form method="POST" action="/new-Drug">
```

Comment:
If you update the url in the WebServlet annotation, don't forget to update this to match.